src/raven.js

```diff
@@ -498,6 +501,35 @@ Raven.prototype = {
         }
     },
 
+    showReportDialog: function (options) {
+        if (!window.document) // doesn't work without a document (React native)
+            return;
+
+        options = options || {};
+
+        var lastEventId = options.eventId || this.lastEventId();
+        if (!lastEventId)
+            return;
+
+        var encode = encodeURIComponent;
+        var qs = '';
+        qs += '?eventId=' + encode(lastEventId);
+        qs += '&dsn=' + encode(this._dsn || '');
+
+        var user = this._globalContext.user;
+        if (user) {
```

Comment:
because the DSN contains the project, and the eventID is bound to the project

often server side project != client side project